source/_integrations/flexpool.markdown

```diff
@@ -0,0 +1,25 @@
+---
+title: Flexpool
+description: Monitor your workers and pool stats in Home Assistant.
+ha_category:
+  - Finance
+ha_release: 2021.02
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@DatDraggy'
+ha_domain: flexpool
+ha_config_flow: true
+---
+
+The Flexpool integration allows you to keep track of your miner details page by adding sensors for workers and pool statistics.
+
+To add the Flexpool sensors to your installation, specify an Ethereum address to watch in the `configuration.yaml` file. You can also optionally provide a token name to retrieve and ERC-20 token balance. If no token is provided then the balance retrieved will be in ETH. You can also optionally provide the token contract address in case the token name is not found.
```

Comment:
- `and ERC-20` -> `and ERC-20

> specify an Ethereum address to watch in the `configuration.yaml` file

I don't think this is correct? As this integration uses a configuration flow, there is no YAML configuration involved.

Reply:
Oh, yea, staged the change but didn't commit. I used the etherscan integration to get a template to work on. Sorry. Fixed.